umm-spec-lib/resources/example-data/iso19115/artificial_test_data.xml

```diff
@@ -1255,6 +1255,7 @@ Distribution liability: NOAA and NCEI make no warranty, expressed or implied, re
                   <gmd:EX_TemporalExtent>
                     <gmd:extent>
                       <gml:TimeInstant gml:id="temporal_extent_3">
+                        <gml:description> | Currentness: Ground Condition</gml:description>
```

Comment:
why the pipe?

Reply:
that is what NOAA has in their data.  No idea why they added it.